.circleci/github-pages.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# This builds docs from current sceptre version and pushes it to the website with lies in separate repository
+set -e
+# show where we are on the machine
+echo "we are in:" $(pwd)
+ls -laF
+# required env vars
+declare -a vars=(
+    CONTAINER_DEST_REPO_DIR # directory in container where dest repo is initialized within container
+    DEST_REPO  # destination repository for rendered pages
```

Comment:
Is this the git url for the docs to be deployed to? If so, then we should probably call it something like: `DEPLOYMENT_URL`.